### PR TITLE
fix: wolf-rbac e2e failed

### PR DESCRIPTION
**Pull request by AlinsRan**
### Type of change:

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

File: test/e2e/testdata/wolf-rbac/start.sh [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 cd testdata/wolf-rbac/
 
+rm -rf db-psql.sql
```
Comment (tao12345666333) [inline test/e2e/testdata/wolf-rbac/start.sh:22]: this is a file, we don't need `-r` argument

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: test/e2e/testdata/wolf-rbac/start.sh (hunk 1 above)
Comment (AlinsRan) [reply test/e2e/testdata/wolf-rbac/start.sh:22]: Need -r, because wolf server may generate a directory with the same name, cause file download failure.

**Review by lingsamuel — APPROVED** [review]

**Comment by tao12345666333** [comment]
re-run all jobs

**Comment by AlinsRan** [comment]
@tao12345666333 The container conflicts. Please help me restart it.

**Pull request closed without merging**